src/components/training/WorkoutReview.jsx

```
@@ -0,0 +1,632 @@
+
+            // Only show completed workouts
+            if (workoutStatus?.status !== "completed") {
+              return null;
+            }
+
+            return (
+              <div
+                key={dayIdx}
+                onClick={() => onViewResults && onViewResults(dayIdx)}
+                className="group cursor-pointer bg-gradient-to-br from-green-900/40 via-emerald-800/50 to-green-900/40 border border-green-500/50 rounded-2xl p-6 hover:shadow-2xl hover:shadow-green-500/20 transition-all duration-300 hover:scale-105"
+              >
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)